bsip-0022.md

```diff
@@ -52,11 +52,11 @@ A [proposal for Peerplays](https://github.com/peerplays-network/pips/blob/master
 
 ## <div id="referendum-categories" /> Referendum Categories
 
-BitShares has four categories of refereundums that core token holders may vote on: witnesses, committee members, worker proposals, and proxy delegation.  Prior to this proposal, any vote within any category applies until changed or until the proxy assignment had been assigned.  The removal of proxy assignment re-instates a core token holders' prior voting selection for witnesses, commitee members.
+BitShares has three categories of referendums that core token holders may vote on: witnesses, committee members, and worker proposals.  Prior to this proposal, any vote within any category applies until changed or until the proxy assignment had been assigned.  The retraction of proxy delegation re-instates a core token holders' prior voting selection for witnesses, commitee members, and worker proposals.
 
-This proposal shall retain the ability of every core token holder to vote on all referendum categories and the optional selection of a single proxy.
+This proposal shall retain the ability of every core token holder to vote on all referendum categories and the optional delegation of a voting proxy.
 
-Vote for three of the referendum categories shall be "decayable": witnesses, committee members, and proxy delegation.  Votes for worker proposals [shall continue to be tabulated as they were prior to this proposal](#out-of-scope) because they are generally fixed in length.
+Votes for two of the referendum categories shall be "decayable": witnesses, committee members.  Votes for worker proposals [shall not be decayable and shall continue to be tabulated as they were prior to this proposal](#out-of-scope) because they are generally fixed in length.  [Vote delegation to a proxy shall be decayable](#proxy-power) [except for votes for worker proposals](#proxied-votes-for-worker-proposals).
```

Comment:
> except for votes for worker proposals

I think this contradicts your clarification in the other commit. Vote delegation *does* decay if the delegator doesn't update. Only if the proxy neglects his duties, the votes will count for workers with 100% and for witnesses/committee with the proxy's decay factor.